commit 6c695a374b0b21a7bb42c53aa0c4238577c2280d
Author: Rosen Penev <rosenp@gmail.com>
Date:   Sat Jun 16 19:17:50 2018 -0700

    staging: mt7621-dts: Fix remaining pcie warnings
    
    [ Upstream commit d0233204fbc10f003d1ef077f57341c2feca4002 ]
    
    This currently fixes the remaining dtb warnings:
    
    Node /pcie@1e140000/pcie0 has a reg or ranges property, but no unit name
    Node /pcie@1e140000/pcie1 has a reg or ranges property, but no unit name
    Node /pcie@1e140000/pcie2 has a reg or ranges property, but no unit name
    Node /pcie@1e140000/pcie0 node name is not "pci" or "pcie"
    Node /pcie@1e140000/pcie0 missing ranges for PCI bridge (or not a bridge)
    Node /pcie@1e140000/pcie0 missing bus-range for PCI bridge
    Node /pcie@1e140000/pcie1 node name is not "pci" or "pcie"
    Node /pcie@1e140000/pcie1 missing ranges for PCI bridge (or not a bridge)
    Node /pcie@1e140000/pcie1 missing bus-range for PCI bridge
    Node /pcie@1e140000/pcie2 node name is not "pci" or "pcie"
    Node /pcie@1e140000/pcie2 missing ranges for PCI bridge (or not a bridge)
    Node /pcie@1e140000/pcie2 missing bus-range for PCI bridge
    Warning (unit_address_format): Failed prerequisite 'pci_bridge'
    Warning (pci_device_reg): Failed prerequisite 'pci_bridge'
    Warning (pci_device_bus_num): Failed prerequisite 'pci_bridge'
    
    device_type was removed since according to documentation, it's deprecated
    for pci(e) devices.
    
    Signed-off-by: Rosen Penev <rosenp@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/mt7621-dts/gbpc1.dts b/drivers/staging/mt7621-dts/gbpc1.dts
index 6b13d85d9d34..87555600195f 100644
--- a/drivers/staging/mt7621-dts/gbpc1.dts
+++ b/drivers/staging/mt7621-dts/gbpc1.dts
@@ -113,6 +113,8 @@
 };
 
 &pcie {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pcie_pins>;
 	status = "okay";
 };
 
diff --git a/drivers/staging/mt7621-dts/mt7621.dtsi b/drivers/staging/mt7621-dts/mt7621.dtsi
index eb3966b7f033..ce6b43639079 100644
--- a/drivers/staging/mt7621-dts/mt7621.dtsi
+++ b/drivers/staging/mt7621-dts/mt7621.dtsi
@@ -447,31 +447,28 @@
 		clocks = <&clkctrl 24 &clkctrl 25 &clkctrl 26>;
 		clock-names = "pcie0", "pcie1", "pcie2";
 
-		pcie0 {
+		pcie@0,0 {
 			reg = <0x0000 0 0 0 0>;
-
 			#address-cells = <3>;
 			#size-cells = <2>;
-
-			device_type = "pci";
+			ranges;
+			bus-range = <0x00 0xff>;
 		};
 
-		pcie1 {
+		pcie@1,0 {
 			reg = <0x0800 0 0 0 0>;
-
 			#address-cells = <3>;
 			#size-cells = <2>;
-
-			device_type = "pci";
+			ranges;
+			bus-range = <0x00 0xff>;
 		};
 
-		pcie2 {
+		pcie@2,0 {
 			reg = <0x1000 0 0 0 0>;
-
 			#address-cells = <3>;
 			#size-cells = <2>;
-
-			device_type = "pci";
+			ranges;
+			bus-range = <0x00 0xff>;
 		};
 	};
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1166_6c695a374b0b21a7bb42c53aa0c4238577c2280d.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1166_6c695a374b0b21a7bb42c53aa0c4238577c2280d.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
